From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Mon, 3 Apr 2017 15:34:03 +0100
Subject: [PATCH] data: Use stateless system directories for d-bus, PAM and
 default rules

[2021-06-03]: In the 0.119 release, the D-Bus policy file is now read from the
              /usr/share location, so we dropped that part of the patch.
[2021-10-04]: Port from autotools to meson.

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 data/meson.build              | 2 +-
 src/polkitbackend/meson.build | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index 073bd98..34fedbb 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -22,7 +22,7 @@ if enable_pam
     output: '@BASENAME@',
     configuration: pam_conf,
     install: true,
-    install_dir: pk_sysconfdir / 'pam.d',
+    install_dir: pk_datadir / 'pam.d',
   )
 endif
 
diff --git a/src/polkitbackend/meson.build b/src/polkitbackend/meson.build
index 64f0e4a..515b171 100644
--- a/src/polkitbackend/meson.build
+++ b/src/polkitbackend/meson.build
@@ -50,7 +50,7 @@ libpolkit_backend = static_library(
 
 install_data(
   '50-default.rules',
-  install_dir: pk_pkgsysconfdir / 'rules.d',
+  install_dir: pk_pkgdatadir / 'rules.d',
 )
 
 program = 'polkitd'
